src/components/TreeReactFlow/Flavor.ts

```diff
@@ -463,7 +463,7 @@ export const flavorLabel = (flavor: NodeFlavor): string => {
     case "TLet":
       return "let";
     case "Pattern":
-      return "P";
```

Comment:
I was just playing with replacing this with "Pattern" &mdash; might as well be explicit about it at beginner level, and we have the space. I agree it could be dropped altogether at higher levels.